### ISSUE TITLE
Idea: Shoutbox

### ISSUE THREAD

**Issue by Kakirtog** [wwevo/guildhome#60]
Add a shoutbox to the main page, which gives the option to shout a short message (100 characters?), displays 5-10 of the last shouts, to make the website more interesting and active.

**Comment by AniOnGit** [comment]
Urm...we already have a huge shoutbox, even called 'shouts' in the middle of the website :D The general problem we currently have is drag users to the website and give them features they wouldn't have otherwise. Noone would use another shoutbox if noone uses normal shouts if you know what I mean ;)

I'd vote to close this.

**Comment by Kakirtog** [comment]
Yeah, maybe you're right. The thing is, there's a bit of a bar to use it, because of 2 reasons I think. First, it gets mixed with the other messages and events, so it feels liek you need to say something big instead of just something randomly fun. Second, it takes you to this separate page to make the shout. I think it's a bit of a difference between a more forum-style thing and just a quick random message.
